<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="01HW198525" name="com.TestSuite.BVT.MyDesk_C1_020" tests="3" failures="2" timestamp="14 Mar 2017 08:14:33 GMT" time="501.688" errors="0">
  <testcase name="MyDesk_C1_020_Method" time="176.411" classname="com.TestSuite.BVT.MyDesk_C1_020">
    <failure type="java.lang.AssertionError" message="null script failed, Wrap Up - null.jpg for more details.">
      <![CDATA[java.lang.AssertionError: null script failed, Wrap Up - null.jpg for more details.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Engine.Reporter.exception(Reporter.java:107)
	at com.WebActions.WebActions.VerifyElementPresentAndClick(WebActions.java:858)
	at com.BusinessModules.MyDesk.Common_Functions.click_on_Wrap_up_button(Common_Functions.java:608)
	at com.TestSuite.BVT.MyDesk_C1_020.MyDesk_C1_020_Method(MyDesk_C1_020.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- MyDesk_C1_020_Method -->
  <testcase name="MyDesk_C1_020_Method" time="165.111" classname="com.TestSuite.BVT.MyDesk_C1_020"/>
  <testcase name="MyDesk_C1_020_Method" time="160.166" classname="com.TestSuite.BVT.MyDesk_C1_020">
    <failure type="java.lang.AssertionError" message="null script failed, Wrap Up - null.jpg for more details.">
      <![CDATA[java.lang.AssertionError: null script failed, Wrap Up - null.jpg for more details.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Engine.Reporter.exception(Reporter.java:107)
	at com.WebActions.WebActions.VerifyElementPresentAndClick(WebActions.java:904)
	at com.BusinessModules.MyDesk.Common_Functions.click_on_Wrap_up_button(Common_Functions.java:608)
	at com.TestSuite.BVT.MyDesk_C1_020.MyDesk_C1_020_Method(MyDesk_C1_020.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:963)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- MyDesk_C1_020_Method -->
</testsuite> <!-- com.TestSuite.BVT.MyDesk_C1_020 -->
